lerna.json

```diff
@@ -1,6 +1,11 @@
 {
   "packages": ["packages/*"],
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
you changed it first to version 1.0 now you change it back, why?

Reply:
Cause when we publish our package with lerna it will automatically update it to proper version depending on our commits messages

Reply:
ok